server/communication/clientCommunication.js

```diff
@@ -80,20 +80,22 @@ const ClientCommunication = {
         Logger.debug('<-- Send Message: ' + messageToSend);
 
         return new Promise((resolve, reject) => {
+            let requestTimeout;
+
```

Comment:
Verursacht das kein Problem, wenn kein Timeout gesetzt ist? Dann würden wir doch clearTimeout of undefined aufrufen?